### PR TITLE
Update cleanup page to use runTests.sh

### PR DESCRIPTION
**Pull request by sypets**
Resolves: #253

### CODE REVIEW

File: Documentation/HandlingAPatch/CleanupTypo3.rst [hunk 1]
```diff
@@ -33,18 +33,38 @@ any case or if in doubt, you can safely perform all steps.
 
 **Flush the cache**:
 
-.. code-block:: shell
+.. tabs::
```
Comment (jonaseberle) [inline Documentation/HandlingAPatch/CleanupTypo3.rst:36]: I think these two are not the same:

`runTests.sh clean` [removes (some) generated files and build cache files](https://github.com/TYPO3/typo3/blob/main/Build/Scripts/runTests.sh#L627-L630) but it does not flush TYPO3 caches.

**Review by jonaseberle — CHANGES_REQUESTED** [review]

File: Documentation/HandlingAPatch/CleanupTypo3.rst (hunk 1 above)
Comment (sypets) [reply Documentation/HandlingAPatch/CleanupTypo3.rst:36]: I think you are right. I will look into it.

**Comment by linawolf** [comment]
@sypets please resolve conflicts

**Comment by sypets** [comment]
Conflicts resolved.

**Pull request merged**